pkgs/development/coq-modules/mathcomp-analysis/default.nix

```
@@ -26,16 +29,19 @@ let
   release."0.2.3".sha256 = "0p9mr8g1qma6h10qf7014dv98ln90dfkwn76ynagpww7qap8s966";
 
   defaultVersion = with versions; lib.switch [ coq.version mathcomp.version ]  [
-      { cases = [ (isGe "8.17") (range "1.15.0" "1.18.0") ];     out = "0.6.6"; }
-      { cases = [ (isGe "8.14") (range "1.15.0" "1.17.0") ];     out = "0.6.5"; }
-      { cases = [ (isGe "8.14") (range "1.13.0" "1.16.0") ];     out = "0.6.1"; }
-      { cases = [ (isGe "8.14") (range "1.13" "1.15") ];         out = "0.5.2"; }
-      { cases = [ (range "8.13" "8.15") (range "1.13" "1.14") ]; out = "0.5.1"; }
-      { cases = [ (range "8.13" "8.15") (range "1.12" "1.14") ]; out = "0.3.13"; }
-      { cases = [ (range "8.11" "8.14") (range "1.12" "1.13") ]; out = "0.3.10"; }
-      { cases = [ (range "8.10" "8.12") "1.11.0" ];              out = "0.3.3"; }
-      { cases = [ (range "8.10" "8.11") "1.11.0" ];              out = "0.3.1"; }
-      { cases = [ (range "8.8"  "8.11") (range "1.8" "1.10") ];  out = "0.2.3"; }
+      { cases = [ (range "8.15" "8.19") (range "2.0.0" "2.2.0")   ]; out = "1.0.0" ; }
```

```suggestion
      { cases = [ (range "8.16" "8.19") (range "2.0.0" "2.2.0")   ]; out = "1.0.0" ; }
```
since mathcomp 2 requires Coq >= 8.16 anyway

---

Ah yes, that was a copy-paste error. Sorry about that.